### PR TITLE
Run console example on Linux

### PR DESCRIPTION
**Pull request by wagenet**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.box = "ubuntu/xenial64"
```
Comment (chancancode) [inline Vagrantfile]: Should we be testing on an older version of Linux?

File: examples/console/Rakefile [hunk 2]
```diff
@@ -1,33 +1,13 @@
...
-
-file "lib/console/native.bundle" => ["lib/console", "cargo:build"] do
-  cp "../../target/release/libconsole.dylib", "lib/console/native.bundle"
-end
-CLOBBER.include("lib/console/native.bundle")
-
-task :irb => "lib/console/native.bundle" do
-  exec "irb -Ilib -rconsole"
-end
+$example_name = 'console'
```
Comment (chancancode) [inline examples/console/Rakefile]: @wycats suggests that you should use `Rake.application.options.example_name = 'console'` instead

File: examples/membership/Rakefile [hunk 3]
```diff
@@ -1,31 +1,14 @@
...
-end
-CLOBBER.include('lib/membership/native.bundle')
-
-task :irb => "lib/membership/native.bundle" do
-  exec "irb -Ilib -rmembership"
-end
-
-task :benchmark => "lib/membership/native.bundle" do
-  exec "ruby -Ilib benchmark.rb"
-end
```
Comment (chancancode) [inline examples/membership/Rakefile]: we should probably keep this around

Comment (wagenet) [inline examples/membership/Rakefile]: Whoops, that was a mistake.

File: examples/console/Rakefile (hunk 2 above)
Comment (wagenet) [inline examples/console/Rakefile]: Are you supposed to set options like that? Anyway, these aren't really options for Rake itself.

File: Vagrantfile (hunk 1 above)
Comment (wagenet) [inline Vagrantfile]: I'll use 14 instead. 12 is almost EOL.

**Pull request merged**